packages/cmd/src/yargs/GroupCommandYargs.ts

```diff
@@ -90,6 +90,7 @@ export class GroupCommandYargs extends AbstractCommandYargs {
                     }
```

Comment:
Since you are already editing this file, should you change line 88, from using the term "Bright" to "Zowe" or "Imperative"?

throw new Error(`Bright Yargs Define Error: Bright command definition type ` +
                                `${child.type} invalid.`);